### PR TITLE
carry lib parameter for branding into catalog show

### LINKED ISSUE #107: [NEOSDiscovery] Institution Branding Converts to NEOS Branding
Different actions that trigger the change:

- Using the exposed Subject headings in the record to browse.
- Letting a record sit idle, and using the previous, next links.

### PR DESCRIPTION
**Pull request by pgwillia**
closes #107

@nnunn I don't recall if you said we should remove the cookie stuff of leave it in.  Let me know what works for you.

### CODE REVIEW

**Comment by pgwillia** [comment]
Hmmm
```
Started POST "/catalog/7603296/track?counter=1&search_id=6" for 127.0.0.1 at 2018-10-11 14:15:27 -0600
Processing by CatalogController#track as HTML
  Parameters: {"authenticity_token"=>"rXuUJFQgbWMDGyNqz0nfi9+TtA8NT5FBFwHvKi/IBvRpDgBWg8nl9YJmYVgIa8NOQ2Nwv+Mbz2mk5K5vR9TIlA=
=", "redirect"=>"http://localhost:3000/catalog/7603296?lib=universityofalberta", "counter"=>"1", "search_id"=>"6", "id"=>"760
3296"}
Redirected to http://localhost:3000/catalog/7603296
```

**Comment by pgwillia** [comment]
@nnunn Think I've got it now.

**Comment by nnunn** [comment]
Great! I can't test it right now, but I'll take a look at it as soon as I get in tomorrow morning. Thanks so much for working on this!

**Comment by nnunn** [comment]
@pgwillia - this looks great and everything is working _except_ the next and previous link on a record page. I took a look, but once again I can't figure out how to add the lib param to those links (these are also in the url_helper_behaviour file. Any chance you could look at that too?

I have removed all the cookies and it's all working perfectly other than the next and previous links.

File: app/views/catalog/_home_text.html.erb [hunk 1]
```diff
@@ -1,49 +1,41 @@
 <div class="row">
   <div class="col-sm-12">
-       <%- if params.has_key?(:lib) %>
-      <%- @library = params[:lib] %>
+    <%- if params.has_key?(:lib) %>
+      <%- @library = $brand %>
```
Comment (pgwillia) [inline app/views/catalog/_home_text.html.erb]: Are the changes below this point just changing the spacing of the file?

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -4,10 +4,7 @@
 			<%= link_to image_tag("neos-logo.jpg"), "https://www.neoslibraries.ca" %>
 		</div>
 		<div class="col-sm-6 footer-links">
-			 <%- if cookies[:brand] %>
-       		 <%- $libraryname = @libraries[cookies[:brand]]["name"] %>
-       		 <%- $homeurl = @libraries[cookies[:brand]]["url"] %>
-       		 <%- $neosurl = @libraries[cookies[:brand]]["neosurl"] %>
+			 <%- if $brand %>
```
Comment (pgwillia) [inline app/views/shared/_footer.html.erb]: I think this might also fix a problem with the email this form.

**Comment by pgwillia** [comment]
@nnunn I think I've found a solution for the previous and next pages.

File: app/views/catalog/_home_text.html.erb (hunk 1 above)
Comment (nnunn) [reply app/views/catalog/_home_text.html.erb]: Yes - sorry about that!

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (nnunn) [reply app/views/shared/_footer.html.erb]: really? how? Gah was my stuff breaking that?

**Comment by nnunn** [comment]
Yes! All works great. I think this is now working perfectly. Thanks again -- you're great!!

**Comment by nnunn** [comment]
I'll let you merge in case you have any further thought/issues/concerns. Do you think we could deploy this early next week? I know they are pushing to have this bug fix up ASAP.

**Pull request merged**